### ISSUE TITLE
[probably not a music file, not adding] Override or Setting Name/Location?

### ISSUE THREAD

**Issue by Tater-K7S5A** [michaelb/auto-ytdl#32]
I downloaded your app yesterday and have begun working with it on my Linux Mint 20.3 system.  

My use case is that I have been downloading audio from some YouTube channel's live streams for off-line listening.  aytdl seems the perfect tool for automating this.

In my experimenting I have gotten [probably not a music file, not adding] output on a couple of things that I have downloaded.  I have looked in the config file and not found any likely suspects for what to change.  I have downloaded several 3 hour music compilations without issue, so I assume it isn't duration related.  'aytdl update --force' will download all the things, so that isn't a workable answer.

Am I missing the setting or is there not one?  If not, can a command be injected in the config to skip whatever the check is that is tripping the [probably not a music file, not adding] action?

Thank you for the application and any assistance you can give me with my use case.

**Comment by michaelb** [comment]
It's been a very long time since I wrote this and both the code and my memory of it are far from perfect

As far as i remember, the [probably not a music file] is triggered by either:

-  a music too short or too long (min and max durations in the config). Beware as some files may report an incorrect lenght to python while looking normal on youtube

- an incorrect file type (allowed extensions also in config)

- empty artists/title fields. You'll have to fiddle a little with that, in the config there should be youtube-dl args that represent the format expected (from the youtube video title -> (artist, title) ). Maybe there is a way to set them correctly for the format of the title of the channels you're targetting


Maybe download one such title directly with youtube-dl and see how long it's and what format?


Another solution, that wouldn't scale/automate well but could get you out of trouble faster sometimes, would be to download separately the problematic sources:

aytdl update --force <url_of_that_one_video>

Then 

aytdl update <url_of_whole_channel>

**Comment by Tater-K7S5A** [comment]
Thank you for responding so quickly.

The streams are as long as 3.5 hours on occasion.  I have been individually downloading them using 'youtube-dl -i --extract-audio --audio-format mp3 --audio-quality 0 https://******'.  Which, obviously, yields an mp3.  

I went into the config file and boosted the 'max _length' to 14400.  I updated 'dateafter' to yesterday and ran aytdl update again.  

It threw '[fromtitle] Could not interpret title of video as "%(artist)s - %(title)s"' at me but it deposited the audio file (as an m4a) in my music directory.  I will have to wait for another channel to update and run it again to see if I have it handled.

If I have further issues I will try fiddling with the artists and titles settings.

Thank you
